x-pack/plugins/monitoring/server/lib/details/get_series.ts

```diff
@@ -193,6 +194,95 @@ async function fetchSeries(
   return await callWithRequest(req, 'search', params);
 }
 
+async function fetchNewSeries(
```

Comment:
same as fetchSeries but creates the indexPattern from `productType` (`elasticsearch`, `kibana`, etc) and no longer accepts `indexPattern`. I didn't want to update all code yet that doesn't need the datastream indices but uses fetchSeries